server/sonar-db-core/src/main/java/org/sonar/db/DatabaseUtils.java

```diff
@@ -333,4 +336,14 @@ public static boolean tableExists(String table, Connection connection) {
   public static IllegalStateException wrapSqlException(SQLException e, String message, Object... messageArgs) {
     return new IllegalStateException(format(message, messageArgs), e);
   }
+
+  public static Consumer<String> setStrings(PreparedStatement stmt, Supplier<Integer> index) {
```

Comment:
It would be great to add a Javadoc explaining that this method can be used in lambda to get rid of the CheckedException